### PR TITLE
update README.md

### PR DESCRIPTION
**Pull request by papeg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,35 +74,45 @@ The loss can theoretically configured in a range from 0.0 to 25.0, but this desi
...
+The aurora core is freerunning and therefore just works. But it can be useful, to check if the connections are up, before running a program, so link configuration errors are easier to detect. For this the [./host/Aurora.hpp](./host/Aurora.hpp) header can be included in your program as a utility. The most important functions are the following.
 
 Create the class. The first parameter is the instance used, either 0 or 1 depending which QSFP port you want to choose. The others needed are the xrt::device and the xrt::uuid of the bitstream.
 
 ```
   Aurora aurora(0, device, xclbin_uuid); 
 ```
 
-The status can be checked with the following function, which waits for the given timeout in ms if the status is ok, which means that the channel is up and alive. If there is an error the function will print the complete aurora core status register.
+This assumes the naming convention used in the example link script. You can also pass the xrt::ip object direclty.
```
Comment (Mellich) [inline README.md:85]:
```suggestion
This assumes the naming convention used in the example link script. You can also pass the xrt::ip object directly.
```

File: README.md [hunk 2]
```diff
@@ -78,35 +74,45 @@ The loss can theoretically configured in a range from 0.0 to 25.0, but this desi
...
-  if (aurora.check_core_status(3000)) {
-    std::cout << "Timeout on startup" << std::endl;
-  } else {
-    std::cout << "Everything is fine" << std::endl;    
-  }
+  Aurora aurora(ip);
 ```
 
-If you are using MPI and each rank controls one aurora core, alternating with the instances - like in the example - you can use the following function to test all links at the same time. If there are errors, the function will print on which cores they happened.
+The status can be checked with the following function, which waits for the given timeout in ms if the status is ok, which means that the channel is up and alive all other status bits are in the wanted state. If there is an error, you can use a function to print the complete aurora core status.
```
Comment (Mellich) [inline README.md:92]: So the function will check the status and wait for 3000ms in the example below? Or will it check the status for 3000ms and return false if the link is still not up after this time?
(Actually, this sounds like a good use case for exceptions but maybe we should postpone this for now)

**Review by Mellich — COMMENTED** [review]
Are the input parameters of our example host application up-to-date?

File: README.md (hunk 2 above)
Comment (papeg) [reply README.md:92]: I will clarify this passage a bit, it's not really clear

**Comment by papeg** [comment]
> Are the input parameters of our example host application up-to-date?

Actually, there is one argument, which wasn't documented all the time. I will add it

**Pull request merged**